charts/datasaur/templates/config/datasaur-llm-lab-tls.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if not .Values.llmTls.secretName }}
+apiVersion: v1
```

Comment:
the file name should be datasaur-llm-labs-tls.yaml

Reply:
bump, this the file name still not changed.


additionally please add the minimum datasaur version requirement @daniel-jeremia